run

```diff
@@ -177,8 +204,8 @@ case $run_command in
 
     # Run watch command in the background
     if [[ ${1:-} == "--watch" ]] || [[ ${1:-} == "-w" ]]; then
-      container_name="yarn-$(date +'%s')"
-      trap 'docker kill ${container_name}' EXIT
```

Comment:
Yeah, this was a misunderstanding from my part, though it did seem broken. It wasn't using `${container_name}` in that scope, and was trying to delete the django container.

I have added more signals to trap and echod the variable directly into the trap command.